x-pack/test/observability_ai_assistant_api_integration/common/users/users.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { kbnTestConfig } from '@kbn/test';
+
+const password = kbnTestConfig.getUrlParts().password!;
+
+export const editorUser: User = {
+  username: 'editor',
+  password,
+  roles: ['editor'],
```

Comment:
Yes, these are the built-in roles that we should aim to use most of the time because those we recommend those to end users.
If needed we can create custom roles to test edge case scenarios.